po/list_translators

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -e -u
+
+SCRIPT_NAME=$(basename "$0")
+
+MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)
+source "$MB_SERVER_ROOT"/script/macos_compat.inc.sh
+
+declare -a OMITTED_AUTHORS=(
+  # From the add-on "Update PO files to match POT (msgmerge)"
+  'Hosted Weblate'
+  # Contractors, avoiding to be self-congratulatory
+  bitmap
+  reosarevok
+  yvanzo
+)
+
+GIT_REVISION_RANGE='origin/production..origin/beta'
+
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME [<git-revision-range>]
+
+List the translators for the given Git revision range.
+
+The default Git revision range is $GIT_REVISION_RANGE
+assuming that this script is called before merging to production.
+
+The following translation authors are intentionally unlisted:
+    $(printf '%s, ' "${OMITTED_AUTHORS[@]}" | sed "s_, \$__")
+
+Commas (and surrounding spaces) in author names are replaced with slashes (/).
+EOH
+)
+
+if [[ $# -gt 1 ]]
+then
+  echo >&2 "$SCRIPT_NAME: too many arguments"
+  echo >&2 "$HELP"
+  exit 64
+elif [[ $# -eq 1 ]]
+then
+  if echo "$1" | grep -Eqx -- '-*h(elp)?'
+  then
+    echo "$HELP"
+    exit
+  else
+    GIT_REVISION_RANGE=$1
+  fi
+fi
+
+cd "$MB_SERVER_ROOT"
+
+git log "$GIT_REVISION_RANGE" -- 'po/*.po' | \
```

Comment:
Should this exclude merges? e.g., `git log v-2024-06-24... -- po/*.po` just returns one merge commit from me (I didn't actually translate anything). And the commit is authored by Michael Wiencek rather than bitmap, so I don't think it would be excluded by `OMITTED_AUTHORS`.

Reply:
Yup. I both added the option `--no-merges` and your full names: https://github.com/metabrainz/musicbrainz-server/compare/7c53cecd5cfde756115e802222b7a7b5618d5f3d..4d988e57657f7c94cfa55ac6d6eb75e6c060160a